build.gradle

```diff
@@ -49,49 +49,11 @@ wrapper.gradleVersion = '7.1.1'
 wrapper.distributionType = Wrapper.DistributionType.ALL
 
 repositories.mavenCentral()
-dependencies.testImplementation(dependencies.platform('org.junit:junit-bom:5.7.2'))
-dependencies.testImplementation('org.junit.jupiter:junit-jupiter:5.7.2')
-
-
-/* ********************* *
- *   Example Programs    *
- * ********************* */
-
-
-sourceSets {
-    example {
-        compileClasspath += sourceSets.main.output
-        runtimeClasspath += sourceSets.main.output
-        output.setResourcesDir(java.destinationDirectory)
-
-        task example(type: Exec) {
-            doFirst {
-                if (!project.hasProperty('toRun')) {
-                    throw new IllegalArgumentException(
-                            'You need to specify which example using the \"toRun\" property!\nE.G. \"./gradlew example -PtoRun=hellofastj\"'
-                    )
-                }
-
-                if (!Files.exists(Path.of(projectDir.toString(), 'src', 'example', 'java', 'tech', 'fastj', 'example', project.getProperty('toRun') as String))) {
-                    throw new IllegalArgumentException(
-                            "The example ${project.getProperty('toRun')} does not exist."
-                    )
-                }
-            }
-
-            dependsOn(tasks.compileExampleJava)
-            description = 'Runs a FastJ example program.'
-            group = 'Execution'
-
-            commandLine(
-                    'java',
-                    '-classpath', sourceSets.example.runtimeClasspath.getAsPath(),
-                    "tech.fastj.example.${project.hasProperty('toRun') ? project.getProperty('toRun') : ''}.Main"
-            )
-        }
-    }
-}
-
+dependencies.testImplementation(dependencies.platform('org.junit:junit-bom:5.8.2'))
+dependencies.testImplementation('org.junit.jupiter:junit-jupiter:5.8.2')
+dependencies.testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+dependencies.api('org.slf4j:slf4j-api:2.0.0-alpha5')
+dependencies.testImplementation('org.slf4j:slf4j-simple:2.0.0-alpha5')
```

Comment:
This is pretty cool and really nice.

Reply:
Yeah! It's definitely been a life saver having a proper logging library.